src/states/elder/mod.rs

```diff
@@ -540,8 +520,10 @@ impl Elder {
             self.sig_accumulator
                 .add_proof(min_section_size, digest, proof)
         {
-            let hop = *self.name(); // we accumulated the message, so now we act as the last hop
-            self.handle_signed_message(signed_msg, route, hop, &BTreeSet::new())?;
+            // we accumulated the message, so now we act as the last hop
+            let mut sent_to = BTreeSet::new();
+            let _ = sent_to.insert(*self.name());
+            self.handle_signed_message(signed_msg, route, &sent_to)?;
```

Comment:
Like in the other comment - `sent_to` is on the chopping block, so it would be nice to find another way of handling this.

Reply:
Understood. This might require a discussion.

Reply:
Here is an idea: we can exclude the sender using their connection info instead of pub_id.

Reply:
We need to filter at the point of choosing the targets, though, so that the number of targets stays correct. If we can map the IDs to connection infos at this point, though, can't we just translate the connection info into the pub_id to be filtered?

Reply:
So reverse lookup the pub_id by the connection info? Sounds doable. Let me try that.